pangeo_forge_orchestrator/models.py

```diff
@@ -0,0 +1,27 @@
+from datetime import datetime
+from typing import Optional
+
+from sqlmodel import SQLModel
+
+from .abstractions import MultipleModels
+
+
+class RecipeRunBase(SQLModel):
+    recipe_id: str
+    run_date: datetime
+    bakery_id: int  # TODO: Foreign key
+    feedstock_id: int  # TODO: Foreign key
+    commit: str
+    version: str
+    status: str  # TODO: Enum or categorical
```

Comment:
Per our conversation today, we are going to punt  🏈  on this for now.

Thanks for copying the full list of check runs fields here, @rabernat. This will be a useful reference for a forthcoming PR.